src/dataMap.js

```diff
@@ -187,10 +187,11 @@ DataMap.prototype.getSchema = function() {
  * @returns {Number} Returns number of created rows.
  */
 DataMap.prototype.createRow = function(index, amount, source) {
-  var row,
-    colCount = this.instance.countCols(),
-    numberOfCreatedRows = 0,
-    currentIndex;
+  let row = null;
```

Comment:
Most of these variables can be moved after `return` or to the body of `if` statement.